### PR TITLE
Fix to segmentation fault issue

### PR DESCRIPTION
**Pull request by avinash-palleti**
Fixed the segementation fault issue while running droneserver, it was due to
conf file path issue.

### CODE REVIEW

File: grpc/server/dronecore_server.cpp.in [hunk 1]
```diff
@@ -49,7 +49,11 @@ void RunServer()
     map_type map;
     std::string plugin;
     std::fstream file;
-    file.open("../proto/plugins.conf");
+    file.open("plugins.conf");
```
Comment (JonasVautherin) [inline grpc/server/dronecore_server.cpp.in]: This depends on where you call `dronecore_server` from, right? Couldn't we do something more like, for instance, `file.open("${PROJECT_SOURCE_DIR}/grpc/proto/plugins.conf");`?

Comment (JonasVautherin) [reply grpc/server/dronecore_server.cpp.in]: Or what about opening `./plugins.conf` by default, but also having it as a parameter?

Comment (JonasVautherin) [reply grpc/server/dronecore_server.cpp.in]: Well, I'm actually realizing it's not part of this "fix" PR. :-)

File: grpc/server/dronecore_server.cpp.in [hunk 2]
```diff
@@ -49,7 +49,11 @@ void RunServer()
...
+    if(!file) {
+        std::cout << "Error in reading conf file" << std::endl;
```
Comment (julianoes) [inline grpc/server/dronecore_server.cpp.in]: What about using `LogErr() <<`?

File: grpc/server/dronecore_server.cpp.in [hunk 3]
```diff
@@ -49,7 +49,11 @@ void RunServer()
     map_type map;
     std::string plugin;
     std::fstream file;
-    file.open("../proto/plugins.conf");
+    file.open("plugins.conf");
+    if(!file) {
```
Comment (julianoes) [inline grpc/server/dronecore_server.cpp.in]: `if(!file) {` -> `if (!file) {`

**Comment by julianoes** [comment]
Let's get this in and fix the small stuff later.

**Pull request merged**